ambry-mysql/src/integration-test/java/com/github/ambry/accountstats/AccountStatsMySqlStoreIntegrationTest.java

```diff
@@ -505,17 +520,8 @@ private static StatsWrapper generateStatsWrapper(int numPartitions, int numAccou
     return TestUtils.generateNodeStats(storeSnapshots, 1000, reportType);
   }
 
-  private void cleanup(MySqlDataAccessor dataAccessor) throws SQLException {
-    Connection dbConnection = dataAccessor.getDatabaseConnection(true);
-    Statement statement = dbConnection.createStatement();
-    for (String table : AccountStatsMySqlStore.TABLES) {
-      statement.executeUpdate("DELETE FROM " + table);
-    }
-  }
-
-  private void assertTableSize(MySqlDataAccessor dataAccessor, int expectedNumRows) throws SQLException {
-    Connection dbConnection = dataAccessor.getDatabaseConnection(true);
-    Statement statement = dbConnection.createStatement();
+  private void assertTableSize(Connection connection, int expectedNumRows) throws SQLException {
+    Statement statement = connection.createStatement();
```

Comment:
I know this is test code, but probably still a good practice to use try-with-resources or try-finally here.